README.md

```diff
@@ -1,981 +1,264 @@
-[![Coverage Status](https://coveralls.io/repos/github/containerbuildsystem/cachito/badge.svg?branch=master)](https://coveralls.io/github/containerbuildsystem/cachito?branch=master)
-
-# Cachito
-
-Cachito is a service to store (and serve) source code for applications. Upon a request, Cachito
-will fetch a specific revision of a given repository from the Internet and store it permanently in
-its internal storage. Namely, it stores the source code for a specific Git commit from a given Git
-repository, which could be from a forge such as [GitHub](https://github.com) or
-[GitLab](https://gitlab.com). This way, even if that repository (or that revision) is deleted, it
-is still possible to track the pristine source code for the original sources. In fact, if the
-sources have already been previously fetched, Cachito will simply serve the stored copy.
-
-Cachito also supports identifying and permanently storing dependencies for certain package managers
-and making them available for building the application. Like it does for source code, future
-requests that utilize these same dependencies will be taken from Cachito's internal storage rather
-than be fetched from the Internet. See the [Package Manager Feature Support](#feature-support)
-section for the package managers that Cachito currently supports.
-
-Cachito will produce bundles as the output artifact of a request. The bundle is a tarball that
-contains the source code of the application and all the sources of its dependencies. For some
-package managers, these dependencies can be used directly for building the application. Other
-package managers will provide an alternative mechanism for this (e.g. a custom npm registry with
-the declared npm dependencies). Regardless of if the dependencies in the bundle are used for
-building the application, they are always present so that the source of these dependencies
-can be published alongside the application for license compliance.
-
-## Table of Contents
-
-* [More Documentation](#more-documentation)
-* [Coding Standards](#coding-standards)
-* [Quick Start](#quick-start)
-* [Pre-built Container Images](#pre-built-container-images)
-* [Prerequisites](#prerequisites)
-* [Development](#development)
-* [Database Migrations](#database-migrations)
-* [API Documentation](#api-documentation)
-* [Configuring Workers](#configuring-workers)
-* [Configuring the API](#configuring-the-api)
-* [Flags](#flags)
-* [Nexus](#nexus)
-* [Package Managers](#package-managers)
+# Cachi2
 
-## More Documentation
+Cachi2 is a CLI tool that pre-fetches your project's dependencies to aid in making your build process hermetic.
```

Comment:
Maybe add a link that explains what [hermetic builds](https://slsa.dev/spec/v0.1/requirements#hermetic) are? Maybe even mention that hermetic builds are meant to secure the build process.

Reply:
Added the link. Trying to mention security had the unfortunate effect of making the line no longer fit on one line, so I decided it wasn't worth it :smile: